Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -543,6 +546,30 @@ void MMU::Memcheck(u32 address, u64 var, bool write, size_t size)
     return;
   }
 
+  // We set up the values for OnMemoryAddressReadFrom and OnMemoryAddressWrittenTo callbacks for
+  // scripts here (which get run in CPU.cpp's run() function later on).
+  if (Scripting::ScriptUtilities::IsScriptingCoreInitialized() && mc->is_enabled &&
+      mc->break_on_hit)
+  {
+    if (write && mc->is_break_on_write)
+    {
+      Scripting::OnMemoryAddressWrittenToCallbackAPI::in_memory_address_written_to_breakpoint =
+          true;
+      Scripting::OnMemoryAddressWrittenToCallbackAPI::
+          memory_address_written_to_for_current_callback = address;
+      Scripting::OnMemoryAddressWrittenToCallbackAPI::
+          value_written_to_memory_address_for_current_callback = var;
+      Scripting::OnMemoryAddressWrittenToCallbackAPI::write_size = static_cast<u32>(size);
+    }
+    if ((!write) && mc->is_break_on_read)
```

Comment:
Do you have any particular reason why the `(!write)` condition is in parenthesis?

If not, the preferred style would be to omit them.
```suggestion
    if (!write && mc->is_break_on_read)
```